### DISCUSSION TITLE
Using hydra with nvim-dap

### DISCUSSION THREAD

**Discussion by ghostbuster91** [scalameta/nvim-metals#567] [Show and tell]
DAP is great but default bindings despite being easy to memorize are cumbersome to use. Of course one could think about different bindings. However, I realized that this is a perfect idea for use of [nvim-hydra](https://github.com/anuvyklack/hydra.nvim/).

And sure enough, someone already did it: https://github.com/anuvyklack/hydra.nvim/issues/3#issuecomment-1162988

I tested it briefly and it works great! Highly recommended.

**Comment by ckipp01** [comment]
Oooo, I wasn't even familiar with nvim-hydra. Thanks for sharing!
